source/math/Quaternion.ooc

```diff
@@ -33,7 +33,7 @@ Quaternion: cover {
 	// z = this y
 	// w = this z
 
-	inverse ::= This new(this w, -this x, -this y, -this z)
+	inverse ::= This new(this w / (this norm * this norm), -this x / (this norm * this norm), -this y / (this norm * this norm), - this z / (this norm * this norm))
```

Comment:
Right, I can change all occurances of inverse on `Quaternion` with conjugate and we'd get code that works the same as before. But, this begs the question, is this **really** what we want? Conjugate and inverse are very different things.